drivers/unifiedPlatform/stworkflows/workflowDataservice.go

```diff
@@ -227,3 +228,29 @@ func (wfDataService *WorkflowDataService) IncreasePvcSizeBy1gb(namespace string,
 	_, err := k8utils.IncreasePVCby1Gig(namespace, deployment, sizeInGb)
 	return err
 }
+
+func (wfDataService *WorkflowDataService) GetDbMasterNode(namespace string, dsName string, deployment map[string]string) (string, bool) {
```

Comment:
@jkagliwal-px could we combine all these four methods and have only one method in the workflow namesd killdbmaster node, which will find and delete the master node of the DB?

Reply:
These where written as one function only, previously but later when scenarios started increaseing, we had to decouple, so that they can be used by different scenarios as well.